ONSdigital/dp-frontend-search-controller
# fixed filter styling and added js and non-js filter form submit

### What

- updated dp-renderer to version to point at new version containing StringArrayContains function
- added functionality from dp-design-system to allow filters to be submitted via js and non-js
- implemented the checkbox component from the design system


### How to review

- pull the `dp-design-system` branch `feature/search-bar` and run locally
- run `dp-frontend-search-controller`
- enter a search and press enter
- the filters down the left hand side of the page should be displayed as expanded
- selecting a child filter should filter the page
- turn of JS, reload the page and the filter button should be visible allowing you to still filter the results
- filter options should persist after the page results refresh or the page is forcefully refreshed
### Who can review

Describe who worked on the changes, so that other people can review.

Anyone
